tgui/packages/tgui/styles/interfaces/Playtime.scss

```diff
@@ -0,0 +1,34 @@
+@use '../base.scss';
+
+.PlaytimeInterface {
+  td {
+    padding: 0;
+    height: 34px;
+  }
+  table {
+    border: 1px solid white;
+    border-radius: 4px;
+  }
+  .Table__cell {
```

Comment:
```
    padding: 0px !important;
```
this is to stop the theme from overriding the cell padding to be massive and making xenos have a massive table in comparison to humans/observer. i forgot this 

(also we should just use a different theme, or move all the hivestatus thing to not be a theme but a stylesheet for that interface... but whatever)